packages/web/src/scss/components/Modal/_theme.scss

```diff
@@ -18,19 +18,27 @@ $common-padding-x-tablet: tokens.$space-800;
 // ModalDialog
 $dialog-width: 640px;
 $dialog-width-desktop: 680px;
-$dialog-height: var(--modal-preferred-height-mobile, min-content);
-$dialog-height-tablet: var(--modal-preferred-height-tablet, min-content);
-$dialog-max-height: min(var(--modal-max-height-mobile, 600px), calc(100dvh - #{2 * $padding-y}));
-$dialog-max-height-tablet: min(var(--modal-max-height-tablet, 600px), calc(100dvh - #{2 * $padding-y}));
+$dialog-height: auto;
+$dialog-height-tablet: auto;
+$dialog-max-height: none;
+$dialog-max-height-tablet: none;
```

Comment:
Maybe we can delete the tablet settings here and the corresponding CSS of `ModalDialog` because there is no change on the tablet breakpoint.